src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/ApplicationSettingsBase.cs

```diff
@@ -726,7 +726,6 @@ private object GetPropertyValue(string propertyName)
                     }
                 }
 
-                object temp = base[propertyName];
```

Comment:
Is it possible this was being used for any kind of argument validation?  There are a variety of checks that result in throwing exceptions in the base method.